docs/api/v1.md

```diff
@@ -417,6 +417,7 @@ Todo....
 {
   id {var id}: Unique identifier of that variable.
   name {string}: Name of the variable as it appears in the function.
+  sTypeIndex {integer}: Index into SourceType array of the type that this variable is.
   sValueIndex {number}: Index to the variable in the SourceValue[] as it is defined in the source code.
```

Comment:
Can't define a single value index for a `SourceVariable`, since they change over the course of a trace, and `SourceVariable`s are part of a function definition. Trace objects refer back to `SourceVariable`s by index instead.